faerie_david/platformio.ini

```diff
@@ -15,8 +15,9 @@ name = Faerie_embedded
 platform = teensy
 board = teensy40
 framework = arduino
-lib_deps = 
-	adafruit/Adafruit NeoPixel@^1.12.0
+lib_deps =
 	ftrias/TeensyThreads@^1.0.2
 	adafruit/Adafruit Unified Sensor@^1.1.14
 	adafruit/Adafruit SHT31 Library@^2.2.2
+	https://github.com/SHC-ASTRA/rover-Embedded-Lib
+	https://github.com/tonton81/FlexCAN_T4
```

Comment:
Do you have to add this here? I thought I remember this dependency, and others, included in rover-Embedded-Lib

Reply:
I wanted to remove all the hard dependencies from the library and instead provide a list of needed dependencies for each project, so that, for example, citadel doesn't need all the CAN and sensor libraries, just Servo.h, so removing those dependencies can free up disk space, make setting up the project locally not take as long on the slow optics lab wifi, and make compile times shorter